IBAnimatable/AnimatableTableViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Created by phimage on 26/03/2017.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+open class AnimatableTableViewController: UITableViewController, ViewControllerDesignable, StatusBarDesignable,
+                                             RootWindowDesignable, TransitionAnimatable, RefreshControlDesignable {
+
+  // MARK: - ViewControllerDesignable
+  @IBInspectable open var hideNavigationBar: Bool = false
+
+  // MARK: - StatusBarDesignable
+  @IBInspectable open var lightStatusBar: Bool = false
+
+  // MARK: - RootWindowDesignable
+  @IBInspectable open var rootWindowBackgroundColor: UIColor?
+
+  // MARK: - TransitionAnimatable
+  @IBInspectable open var _transitionAnimationType: String? {
```

Comment:
I think we don't want that to be `open` and let it only visible in Interface Builder. We can remove `open` and let it as  the default `internal`.